File: iFlat/src/main/java/com/iflat/sm/bean/QualityFine.java
package com.iflat.sm.bean;

import java.util.Date;

/**
 * Created by tyriv on 2016/1/7.
 */
public class QualityFine {

    private String id;
    private String projNo;  //工号
    private Date date;  //日期
    private String dept;  //部门
    private String team;  //施工队
    private String group;  //施工班组
    private String personAcc;  //人员账号
    private String personName;  //人员姓名
    private String profession;  //专业
    private String description;  //描述
    private String category;  //性质
    private double amount;  //金额
    private String comment;  //备注
    private String qc;  //考核人
    private String creator;  //录入人
    private Date createTime;  //录入日期
    private String attachment;  //附件

    public String getAttachment() {
        return attachment;
    }

    public void setAttachment(String attachment) {
        this.attachment = attachment;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public double getAmount() {
        return amount;
    }

    public void setAmount(double amount) {
        this.amount = amount;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getProjNo() {
        return projNo;
    }

    public void setProjNo(String projNo) {
        this.projNo = projNo;
    }

    public String getDept() {
        return dept;
    }

    public void setDept(String dept) {
        this.dept = dept;
    }

    public String getTeam() {
        return team;
    }

    public void setTeam(String team) {
        this.team = team;
    }

    public String getGroup() {
        return group;
    }

    public void setGroup(String group) {
        this.group = group;
    }

    public String getPersonAcc() {
        return personAcc;
    }

    public void setPersonAcc(String personAcc) {
        this.personAcc = personAcc;
    }

    public String getPersonName() {
        return personName;
    }

    public void setPersonName(String personName) {
        this.personName = personName;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public String getProfession() {
        return profession;
    }

    public void setProfession(String profession) {
        this.profession = profession;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getComment() {
        return comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }

    public String getQc() {
        return qc;
    }

    public void setQc(String qc) {
        this.qc = qc;
    }

    public String getCreator() {
        return creator;
    }

    public void setCreator(String creator) {
        this.creator = creator;
    }

    public Date getCreateTime() {
        return createTime;
    }

    public void setCreateTime(Date createTime) {
        this.createTime = createTime;
    }
}

File: web/app/model/qm/QualityFine.js
Ext.define('iFlat.model.qm.QualityFine', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'qualityFine.id', mapping: 'id', type: 'string'},
        {name: 'qualityFine.projNo', mapping: 'projNo', type: 'string'},
        {name: 'qualityFine.projName', mapping: 'projName', type: 'string'},
        {name: 'qualityFine.dept', mapping: 'dept', type: 'string'},
        {name: 'qualityFine.team', mapping: 'team', type: 'string'},
        {name: 'qualityFine.group', mapping: 'group', type: 'string'},
        {name: 'qualityFine.personAcc', mapping: 'personAcc', type: 'string'},
        {name: 'qualityFine.personName', mapping: 'personName', type: 'string'},
        {name: 'qualityFine.date', mapping: 'date', type: 'date'},
        {name: 'qualityFine.profession', mapping: 'profession', type: 'string'},
        {name: 'qualityFine.description', mapping: 'description', type: 'string'},
        {name: 'qualityFine.category', mapping: 'category', type: 'string'},
        {name: 'qualityFine.amount', mapping: 'amount', type: 'number'},
        {name: 'qualityFine.comment', mapping: 'comment', type: 'string'},
        {name: 'qualityFine.qc', mapping: 'qc', type: 'string'},
        {name: 'qualityFine.attachment', mapping: 'attachment', type: 'string'},
        {name: 'qualityFine.creator', mapping: 'creator', type: 'string'},
        {name: 'qualityFine.createTime', mapping: 'createTime', type: 'date'},
        {name: 'qualityFine.score', mapping: 'score', type: 'number'},
    ]
});